roles/ceph-defaults/defaults/main.yml

```diff
@@ -425,6 +425,10 @@ email_address: foo@bar.com
 # Monitor handler checks
 handler_health_mon_check_retries: 5
 handler_health_mon_check_delay: 10
+
+# Monitor protocols
+mon_host_v1_port: "6789"
```

Comment:
docs suggest this should be sufficient to change the binding port [1]

1. http://docs.ceph.com/docs/master/rados/configuration/msgr2/

Reply:
`mon host` is not used to control the port binding. it's only use to know how we can contact the mons member.
this probably need to be done via `mon addr` variable under `[mon.<name>`] section

Reply:
According to this doc section [1] I think in Nautilus you can use just the ip address if you want to bind the default ports, but the new syntax allows users to specify different port numbers (both v1 and v2), so using this variables we'll add more flexibility from a user point of view.

1. http://docs.ceph.com/docs/master/rados/configuration/msgr2/#updating-ceph-conf-and-mon-host/

Reply:
well the doc is one thing, did you try it (with different port value) ? because on my side it's not working as you describe.
again `mon host` isn't for por binding configuration.